src/Listener/Traits/FormTypeTrait.php

```diff
@@ -56,33 +56,41 @@ protected function _getFormSubmitExtraButtons()
             return [];
         }
 
-        $defaults = [
-            [
+        if ($buttons === null || $buttons === true) {
+            $buttons = $this->_getDefaultExtraButtons();
+        }
+
+        return $buttons;
+    }
+
+    /**
+     * Get default extra buttons
+     *
+     * @return array
+     */
+    protected function _getDefaultExtraButtons()
+    {
+        return [
+            'save_and_continue' => [
                 'title' => __d('crud', 'Save & continue editing'),
-                'options' => ['class' => 'btn btn-success btn-save-continue', 'name' => '_edit', 'value' => true],
+                'options' => ['class' => 'btn btn-success btn-save-continue', 'name' => '_edit', 'value' => '1'],
                 'type' => 'button',
                 '_label' => 'save_and_continue',
             ],
-            [
+            'save_and_create' => [
```

Comment:
This seems like it might be backwards incompatible?

Reply:
I highly doubt someone would have been relying on the numeric indexes. Also this list of defaults was moved from template to the this trait pretty recently so it's highly unlikely to cause BC problems.

Reply:
Would it break the existing form element?

Reply:
Nope it won't. This array is only used in `Element/form/buttons.ctp` and this change won't cause any problem.